feat: daily challenges 30-54 (#61991)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/daily-coding-challenges-javascript/68af0687ef34c76c28ffa54d.md

---
id: 68af0687ef34c76c28ffa54d
title: "JavaScript Challenge 33: Screen Time"
challengeType: 28
dashedName: javascript-challenge-33
---

# --description--

Given an array of seven integers, where each integer represents the amount of hours spent on your phone each day for a week, determine if it is too much screen time based on these constraints:

> - If any one day has 10 hours or more, it's too much.

Author: I'm not sure I like starting with "The input is" - I've been starting pretty much all challenges with "Given..." - maybe "Given an array input of seven integers..."?

I also haven't been using function names in the descriptions - e.g: `tooMuchScreenTime` - reason is that the description is shared between JS and Python (planning to add a curriculum test for it, right now it's only checked when seeding the db) - and in python, the function names are a little different - e.g: `too_much_screen_time` is convention in python to my understanding - so I've been using that.

> Given an array of seven integers, where each integer represents the amount of hours spent on your phone each day for a week, determine if it is too much screen time based on these constraints:
> 
> - If any one day has 10 hours or more, it's too much.

Reviewer: Sure, that all makes sense. How about this:
```suggestion
Given an array of seven integers, where each integer represents the amount of hours spent on your phone in one of the days of the week, determine if it is too much screen time based on these constraints:

- If any day has 10 hours or more, it's too much.
```
